pipelines/main/launch_unsigned_jobs.yml

```diff
@@ -12,6 +12,25 @@
 # and only need to touch the webui configuration when we need to alter
 # something about the privileged steps.
 
+common:
+  - diff-filter-build_plugin: &diff-filter-build
+      https://github.com/fatteneder/diff-filter-buildkite-plugin#main:
+        name: "build"
+        ignore:
+          - "*.md"
+          - "*.json"
+          - "CITATION.*"
+          - ".gitignore"
+          - ".clangd"
+          - ".mailmap"
```

Comment:
What does this ignore-list?  Can we please have comments to explain how the filtering logic works?

Reply:
I added an explanation to the plugin's readme: https://github.com/fatteneder/diff-filter-buildkite-plugin